workers/src/main/java/edu/cornell/testenv/testrunner/JUnit5TestRunner.java

```diff
@@ -32,7 +32,7 @@ public class JUnit5TestRunner implements TestRunner {
     @Override
     public boolean runTest(TestEnvContext context, TestOutputStream outputStream, File rootDir) {
         try {
-            //Assumption is class files live in /build/classes/java/
+            //Assumption is class files live in /build/classes/java/ (and the .class files in some subdirectory)
             String baseRootPath = rootDir.toString() + "/build/classes/java/".replaceAll("/", File.separator);
```

Comment:
Should this be included in the config so we don't have to hardcode it? Pretty sure we already have the list of test directories in the config

Reply:
Ideally yes. Either in this PR or a separate small PR.

Reply:
Let's do separate PR